.travis.yml

```diff
@@ -5,10 +5,11 @@ language: python
 # Enable 3.7 without globally enabling 'dist: xenial' for other build jobs
 matrix:
   include:
-    - python: 3.5
     - python: 3.6
     - python: 3.7
       dist: xenial
+    - python: 3.8
+      dist: xenial
```

Comment:
It's better to always use Xenial, which I think has become the default some time ago. See [pyccel](https://github.com/pyccel/pyccel/blob/master/.travis.yml), or maybe even better [sympde](https://github.com/pyccel/sympde/blob/master/.travis.yml).